### DISCUSSION TITLE
Scrape paid-for and free messages from non-active models

### DISCUSSION THREAD

**Discussion by fragme11** [UltimaHoarder/UltimaScraper#1603] [Q&A]
I have a few models that I have paid content for, but since I no longer subscribe to them, I'm not sure how to scrape the files. I can still access the files on the site's messaging feature. Is there a way to do it?

Thanks!

**Comment by Nick05491** [comment]
Bump... also interested in knowing how to do this.

**Comment by ilovescraperz** [comment]
Has there been an update on this? Bought a couple vids in messages from a model i'm not currently subbed to and hoping to scrape it without subbing....
